### DISCUSSION TITLE
We used Open MCT on the cloud to make this...

### DISCUSSION THREAD

**Discussion by milesbjackson** [nasa/openmct#6392] [Show and tell]
So we've modified Open MCT to run on the cloud, amongst a few other details. (GitHub forthcoming when i can finish pulling it together!) In this video we pipe 60,000 data points per second from the game playseat here in Milton Keynes, UK. The data goes to an Open MCT Telemetry server and an openmct server running in North Carolina. We then serve the data back as a web client to a laptop at Milton Keynes and project it on the wall behind Max Verstappen to make these videos.

This is recorded live. No gimmicks. Look how amazing Open MCT is. Almost zero latency! This is across an ocean and back again.

https://youtu.be/RATD9Hb04o4

**Comment by akhenry** [comment]
@milesbjackson  This is incredible, thanks for sharing this with us! It's so cool to see Open MCT being used in an application like this. Was this a one off, or are you using Open MCT more regularly?

**Comment by milesbjackson** [comment]
Andrew,

I’m using it more and more!

My main goal is to have it running fast telemetry on the cloud and this is working very well!

We are building more and more widgets and expanding the telemetry API and additional clocks etc. Adding in computed ‘live’ annotations to plots etc.

Lots to do! But aim is to have GitHub available so people can pick it up pieces and use in their applications. Add into the plugin library for people to use.

Just loving how well it works!

Miles

From: Andrew Henry ***@***.***>
Date: Monday, 6 March 2023 at 18:13
To: nasa/openmct ***@***.***>
Cc: Miles Jackson ***@***.***>, Mention ***@***.***>
Subject: [External] : Re: [nasa/openmct] We used Open MCT on the cloud to make this... (Discussion #6392)
The message from this sender included one or more files
which could not be scanned for virus detection; do not
open these files unless you are certain of the sender's intent.
________________________________

@milesbjackson<https://urldefense.com/v3/__https:/github.com/milesbjackson__;!!ACWV5N9M2RV99hQ!ILXuf1VVQ8h3cWSekQUCbSpKrMg1aszzP13714T84EQg0SHvyWr-K_loVdMcvehn84YKsYYayNrb6c2l5dyYrYFXNWI$> This is incredible, thanks for sharing this with us! It's so cool to see Open MCT being used in an application like this. Was this a one off, or are you using Open MCT more regularly?

**Comment by simnaut** [comment]
Awesome Miles! I am interested in learning more about how you got it set up for speed on the cloud.

**Comment by lalligagger** [comment]
This is awesome. How do I quit my job as a satellite engineer and get on the F1 circuit?
